src/processing/IDrawShapes.java

```diff
@@ -0,0 +1,28 @@
+package processing;
+
+import processing.utility.*;
+
+/**
+*
+* @author Himanshu Jain
+*/
+
+public interface IDrawShapes {
+    // parameter: Pixel center - center position with intensity for circle
+    //            radius - radius of circle
+    void draw_circle(Pixel center, float radius);
```

Comment:
[Coding convention for methods](https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names
)
Please refer to the coding conventions and do necessary changes.